app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -338,7 +338,8 @@ private int getSelectedLogo() {
 
         if (mUsePhoneProvider.isChecked()) {
             selectedProviders.add(
-                    new IdpConfig.Builder(AuthUI.PHONE_VERIFICATION_PROVIDER).build());
+                    new IdpConfig.Builder(AuthUI.PHONE_VERIFICATION_PROVIDER)
+                            .build());
```

Comment:
Might as well undo the changes in this file since they're leftovers from previous commits.

Reply:
Reverted.